compute_endpoint/globus_compute_endpoint/MULTIUSER_SETUP.md

```diff
@@ -0,0 +1,334 @@
+# Multi-User Compute Endpoints
+
+Multi-user compute endpoints (MEP) enable an administrator to securely offer
+their compute resources to their users without mandating the typical shell
+(i.e., ssh) access.  Administrators may preset endpoint configuration options
+(e.g., Torque, PBS, or Slurm), and also offer user-configurable items (e.g.,
+account id, problem-specific number of blocks).
+
+## tl;dnr
```

Comment:
Och; that changed since it first entered the collective lexicon.  It was specifically "Too long; did not read!" when originally discussed -- evidence is "I was there" in various internet circles.  Though both were acceptable at the time, the tl;dnr was more common and generally preferred -- it was as a joke to be "formal" (because no contractions!) ... in an obviously informal phrase.  (As opposed to "Too long; didn't read!")

For reference, [note that Wikipedia accepts _both_](https://en.wikipedia.org/w/index.php?title=TL;DNR&redirect=no), but discusses only TL;DR.

Meanwhile, I see that OED has decided that "Too long; didn't read" is the winner.  Alas.

Reply:
dang natural language, always pulling the rug out from under you

Reply:
:boom: